src/hooks/sdk/docs/useMediaStreamMetrics.stories.mdx

```diff
@@ -0,0 +1,62 @@
+<Meta title="SDK Hooks/useBandwidthMetrics" />
```

Comment:
Can you update the title here :) ?

Reply:
God catch. Will do.

Reply:
Should not resolve the comment before it is fixed.